README.md

```diff
@@ -1,13 +1,12 @@
 <p align="center">
-  <a href="https://www.runpanther.io"><img src=".img/panther-logo-github-highres.png" width=75% alt="Panther Logo"/></a>
+  <a href="https://www.runpanther.io"><img src=".img/panther-logo-github-highres.png" width=40% alt="Panther Logo"/></a>
```

Comment:
idk if this is really necessary but should we move all of these links to panther.com?

Reply:
Good catch,  updated along with a couple doc links I missed